operators/index.js

```diff
@@ -5,6 +5,18 @@ function toBuffer(value) {
   return Buffer.isBuffer(value) ? value : Buffer.from(value)
 }
 
+function key(value) {
+  return {
+    type: 'EQUAL',
+    data: {
+      seek: seekers.seekKey,
+      value: Buffer.from(value),
+      indexType: 'key',
+      prefix: 32
+    },
+  }
+}
```

Comment:
TODO in this file: convert the implementation of these to use `jitdbOperators.equal`, see also `type` and `author` below. Not strictly needed for this PR, though. The benefit is that we move away the details of the shape of the object to jitdbOperators.equal, because that's the function responsible for this

Reply:
created an issue for that